src/test/java/posters/pageobjects/pages/checkout/PaymentPage.java

```diff
@@ -31,9 +31,10 @@
 
     @Override
     @Step("ensure this is a payment page")
-    public void isExpectedPage()
+    public PaymentPage isExpectedPage()
     {
         headline.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.